spec/lib/submission_builder/ty2022/states/ny/individual_return_spec.rb

```diff
@@ -107,7 +107,7 @@
       it "creates an additional dependents pdf" do
         submission_builder = SubmissionBuilder::Ty2022::States::Ny::IndividualReturn.new(submission)
         additional_dependents = submission_builder.pdf_documents.select do |d|
-          d.pdf == PdfFiller::AdditionalDependentsPdf
+          d.pdf == SubmissionBuilder::Ty2022::States::Ny::Documents::It201AdditionalDependents
```

Comment:
should we check that this doesn't get added when there are <= 6 dependents? maybe overkill since pdfs are less important than xml, so up to you

Reply:
Good call.